### ISSUE TITLE
[BUG] Consistency error: Note XXX content is null even though it is not deleted

### ISSUE THREAD

**Issue by nil0x42** [zadam/trilium#696]
version: 0.36.3

A consistency error message keeps showing regularly:
```
Consistency error: Note XXX content is null even though it is not deleted
```

The blamed note is a book note type, i don't remember buy it might have been of another type before.
Anyway, as it's a book, having a null content should not be a problem, i don't know what happened.

I have other books, but the consistency error only appears for this specific one.

**Comment by zadam** [comment]
Hello, I think this is related to #694 - I managed to reproduce this on 0.36.3 (before #694 has been fixed) by first having text note with children and empty content (so in "auto-book" mode) and then I converted to "real" book type. I'm just releasing 0.36.4 which should fix this issue.

However to fix your DB, open SQL console (ALT-O) and execute:
```sql
UPDATE note_contents SET content = '' WHERE noteId = 'XXX' AND content IS NULL
```

These consistency checks are sometimes bit strict - as you say book note could be null, but I put a logical constraint that non-deleted notes must be always non null - such checks then manage to reveal some unexpected handling of data and thus reveal bugs.
